clients/go/coreutils/typing.go

```diff
@@ -0,0 +1,140 @@
+package coreutils
```

Comment:
Same question here, the naming of the file does not describe what this module is doing